omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/ParserEncounterIntoSimpleObjectsTest.java

```diff
@@ -297,6 +300,29 @@ public void testParsingSimpleObs() throws Exception {
 		assertThat(path(parsed.getObs(), 0, "answer"), is((Object) valueText));
 	}
 
+	@Test
+	public void testParsingNumericDataTypeWithAllowDecimalAsFalseToSimpleObs() throws Exception {
```

Comment:
`testParsingNumericObsShouldHandleAllowDecimal`

It would be great to test also the case when allo decimal is `true`.

Reply:
Sure, I will create another test for that case.